apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/BlockchainProcessorImpl.java

```diff
@@ -1582,7 +1631,12 @@ private void downloadPeer() throws InterruptedException {
                     log.debug("Null response wile getCumulativeDifficultyRequest from peer {}",peer.getHostWithPort());
                     return;
                 }
-                BigInteger curCumulativeDifficulty = lookupBlockhain().getLastBlock().getCumulativeDifficulty();
+                Block lastBlock = lookupBlockhain().getLastBlock();
+                if (lastBlock == null) {
```

Comment:
Looks like its not possible

Reply:
In fact, it's happening with sharded DB.

Reply:
removed